src/dbnode/storage/index/convert/convert.go

```diff
@@ -188,6 +190,76 @@ func FromSeriesIDAndTagIter(id ident.ID, tags ident.TagIterator) (doc.Metadata,
 	return d, nil
 }
 
+// FromSeriesIDAndEncodedTags converts the provided series id and encoded tags into a document.
```

Comment:
Nit - might be more appropriate after rename:
```suggestion
// FromSeriesIDAndEncodedTags converts the provided series id and encoded tags into a doc.Metadata.
```